tools/karpenter-convert/pkg/convert/testdata/nodeclass_kubectl_output.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: karpenter.k8s.aws/v1beta1
+kind: EC2NodeClass
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
```

Comment:
Is this right? I'm not sure that this annotation appears when we just run the conversion tool

Reply:
Any annotation provided in input would be available in output. For clarity I removed the annotations from the testdata.